symfony.lock

```diff
@@ -172,6 +172,15 @@
             "ref": "b4edd2a1e6ac1827202f336cac2771cb529de542"
         }
     },
+    "sonata-project/doctrine-extensions": {
```

Comment:
I see that `symfony.lock` was already in git. So let it be then and do not remove it (so opposite what I have previously wrote on Slack)